src/internal/common/str/str.go

```diff
@@ -0,0 +1,58 @@
+package str
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/alcionai/clues"
+
+	"github.com/alcionai/corso/src/internal/common/ptr"
+)
+
+// parseBool returns the bool value represented by the string
+// or false on error
+func ParseBool(v string) bool {
+	s, err := strconv.ParseBool(v)
+	if err != nil {
+		return false
+	}
+
+	return s
+}
+
+func FromMapToAny(k string, m map[string]any) (string, error) {
```

Comment:
maybe this should have a different name that denotes that it returns a string? There's now two functions named `FromMapToAny`, one in this package and one in `tform`. They're not functions that adhere to an interface (different return types) so it can be a little confusing. It can also confuse GitHub's search and simpler search tools like grep

Reply:
can do